### ISSUE TITLE
Oauth2 / Invalid class Tokeinfo after migrating namespaces

### ISSUE THREAD

**Issue by davaxi** [googleapis/google-api-php-client-services#392]
```php
<?php
//...

$oauth = new \Google_Service_Oauth2($clien);
$ouath->tokeninfo();

// >> Exception : Class 'Google\Service\Tokeninfo' not found
```

Add in : https://github.com/googleapis/google-api-php-client-services/blob/ee1072221acf7c32e3de9b18e11fec3ab23ec38f/src/Oauth2.php#L20
```php
<?php

//...

use Google\Service\Oauth2\Tokeninfo;

//...
```

**Comment by MichaelBelgium** [comment]
How is this not fixed yet, its a simple imporrt? (on v0.224.0)

**Comment by bshaffer** [comment]
Fixed in https://github.com/googleapis/google-api-php-client-services/pull/553
